Filter/BooleanFilter.php

```diff
@@ -32,23 +32,35 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
                     continue;
                 }
 
-                $values[] = ($v == BooleanType::TYPE_YES) ? 1 : 0;
+                $values[] = ($v == BooleanType::TYPE_YES) ? true : false;
```

Comment:
Could you change that to simply : `$values[] = ($v == BooleanType::TYPE_YES)` ? It gives the same result.